Dockerfile

```diff
@@ -78,14 +78,14 @@ RUN yum -y install --setopt=tsflags=nodocs ruby-install make
 # Install Ruby 2.3.1
 RUN ruby-install ruby 2.3.1 -- --disable-install-doc && rm -rf /usr/local/src/* && yum clean all
 
-## Build Auth-Api
-ENV HTTPD_AUTH_API_SERVICE_DIRECTORY=/opt/auth-api
-RUN mkdir -p ${HTTPD_AUTH_API_SERVICE_DIRECTORY}
-COPY docker-assets/auth-api ${HTTPD_AUTH_API_SERVICE_DIRECTORY}
-RUN  cd ${HTTPD_AUTH_API_SERVICE_DIRECTORY} && \
-     gem install bundler && \
-     bundle install
-COPY docker-assets/auth-api.service    /usr/lib/systemd/system/auth-api.service
+## Install DBus API Service
+ENV HTTPD_DBUS_API_SERVICE_DIRECTORY=/opt/dbus-api-service
+RUN mkdir -p ${HTTPD_DBUS_API_SERVICE_DIRECTORY}
+RUN cd ${HTTPD_DBUS_API_SERVICE_DIRECTORY} && \
+    curl -L https://github.com/ManageIQ/dbus_api_service/tarball/master | tar vxz -C ${HTTPD_DBUS_API_SERVICE_DIRECTORY} --strip 1 && \
```

Comment:
Eventually we'll want to release this...I'm wondering if it's better to make the version an ENV var, then use the version here.  That way it's easier to eyeball what version we are using.